Sources/FrontEnd/Parse/Parser.swift

```diff
@@ -133,6 +133,14 @@ public enum Parser {
               appearsBeforeAccessModifier: access))
         }
 
+        // Catch incosistent access modifiers.
+        else if let prev = accessModifiers.first, prev != access {
+          state.diagnostics.insert(
+            .error(
+              inconsistentAccessModifiers: access,
+              appearsAfterPreviousAccessModifier: prev))
+        }
```

Comment:
```suggestion
        else if let p = accessModifiers.first, p != access {
          state.diagnostics.insert(
            .error(
              inconsistentAccessModifiers: access,
              appearsAfterPreviousAccessModifier: p))
        }
```

We tend to avoid abbreviations in the code, or use single letter variables if the context allows.